src/components/my-career/SkillsRecommendationCourses.jsx

```diff
@@ -0,0 +1,130 @@
+import React, {
+  useContext, useMemo, useState, useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { AppContext } from '@edx/frontend-platform/react';
+import { SearchContext } from '@edx/frontend-enterprise-catalog-search';
+import { CardGrid, Hyperlink } from '@edx/paragon';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import SearchCourseCard from '../search/SearchCourseCard';
+
+import { useDefaultSearchFilters, useSearchCatalogs } from '../search/data/hooks';
+import { UserSubsidyContext } from '../enterprise-user-subsidy';
+import { SubsidyRequestsContext } from '../enterprise-subsidy-requests';
+
+const SkillsRecommendationCourses = ({ index, subCategoryName, facetSkills }) => {
+  const { enterpriseConfig } = useContext(AppContext);
+  const {
+    subscriptionPlan, subscriptionLicense, couponCodes: { couponCodes }, enterpriseOffers,
+  } = useContext(UserSubsidyContext);
+  const { catalogsForSubsidyRequests } = useContext(SubsidyRequestsContext);
+
+  const searchCatalogs = useSearchCatalogs({
+    subscriptionPlan,
+    subscriptionLicense,
+    couponCodes,
+    enterpriseOffers,
+    catalogsForSubsidyRequests,
+  });
+
+  const { filters } = useDefaultSearchFilters({
+    enterpriseConfig,
+    searchCatalogs,
+  });
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [courses, setCourses] = useState([]);
+  const [hitCount, setHitCount] = useState(undefined);
+  const { refinements } = useContext(SearchContext);
+  const { skill_names: skills } = refinements;
```

Comment:
I don't think this change is needed, we don't append any skills in url on career tab. We just need the skills relevant to the sub-category.